cmd/client-fs.go

```diff
@@ -765,33 +702,9 @@ func (f *fsClient) listInRoutine(contentCh chan<- *clientContent, isMetadata boo
 			if fi.Mode()&os.ModeSymlink == os.ModeSymlink {
 				fp := filepath.Join(fpath, fi.Name())
 				fi, e = os.Stat(fp)
-				if os.IsPermission(e) {
-					contentCh <- &clientContent{
-						Err: probe.NewError(PathInsufficientPermission{Path: pathURL.Path}),
-					}
-					continue
-				}
-				if os.IsNotExist(e) {
```

Comment:
We added this 
```
if os.IsNotExist(e) {
	// Lstat makes no attempt to follow the broken link.
	_, e = os.Lstat(fp)
	contentCh <- &clientContent{
		URL:  *newClientURL(fp),
		Size: -1,
		Err:  probe.NewError(e),
	}
	return nil
}
``` 
to ignore any broken symlink during mirroring/listing. else mirror would break in the middle if it sees any broken link while listing

Reply:
That assumption is no longer true, because the mirror ignores all broken symlinks automatically - we simply have to keep continuously list.

Reply:
No @harshavardhana, mirroring will fail here in the first catch if it sees any broken link. Will not proceed further mirroring the remaining uninfected files. Is this ok?

Reply:
For eg,

```
touch 1{01..10}
ln -svf nonexisting 105
mc mb play/test
mc mirror . play/test
```

Reply:
This should be fixed,